docs/manual.txt

```diff
@@ -529,6 +529,30 @@ the `markdown metadata extension docs <https://pythonhosted.org/Markdown/extensi
 
 Note that keys are converted to lowercase automatically.
 
+HTML metadata
```

Comment:
add it after line 420; I think *HTML meta tags* would be a more descriptive name.